### ISSUE TITLE
VLC addon not working on hard reboot HA OS 7.5

### ISSUE THREAD

**Issue by Efendis** [home-assistant/addons#2416]
### Describe the issue you are experiencing

VLC addon not working on hard reboot HA OS 7.5 or after turning off/on the PC

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Which add-on are you reporting an issue with?

VLC

### What is the version of the add-on?

0.1.3

### Steps to reproduce the issue

1. Restart Home assistant OS 7.5
2.
3.
...

### Anything in the Supervisor logs that might be useful for us?

```txt
none
```

### Anything in the add-on logs that might be useful for us?

```txt
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f9651ca33a0] main decoder error: buffer deadlock prevented
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
[00007f965a300d50] vlcpulse audio output error: stream connection failure: Bad state
[00007f965a300d50] main audio output error: module not functional
[00007f9651ca33a0] main decoder error: failed to create audio output
```

**Comment by MartinHjelmare** [comment]
Does restarting the add-on solve the issue?

**Comment by Efendis** [comment]
> Does restarting the add-on solve the issue?

Yes.

**Comment by Efendis** [comment]
After reboot add-on:
Logger: homeassistant.components.vlc_telnet
Source: components/vlc_telnet/media_player.py:94
Integration: VLC media player via Telnet (documentation, issues)
First occurred: 17:39:35 (1 occurrences)
Last logged: 17:39:35

Connection error: Failed to read: 0 bytes read on a total of undefined expected bytes. Partial bytes read: b''

**Comment by MartinHjelmare** [comment]
Does the integration reconnect successfully? Ie can you use the integration after restarting the add-on and allowing the integration to reconnect?

**Comment by Efendis** [comment]
After restarting the addon, it works fine.

**Comment by MartinHjelmare** [comment]
What board/machine are you using for Home Assistant?

**Comment by Efendis** [comment]
Hi. Hardware: Beelink U59

**Comment by MartinHjelmare** [comment]
Thanks! Please copy the System Health information from the Configuration Info tab and paste in a comment.

**Comment by Efendis** [comment]
## System Health

version | core-2022.3.5
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.9.9
os_name | Linux
os_version | 5.10.103
arch | x86_64
timezone | Europe/Samara

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 4933
Installed Version | 1.23.0
Stage | running
Available Repositories | 1074
Downloaded Repositories | 11

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 7.5
-- | --
update_channel | stable
supervisor_version | supervisor-2022.03.5
docker_version | 20.10.9
disk_total | 237.7 GB
disk_used | 15.7 GB
healthy | true
supported | true
board | generic-x86-64
supervisor_api | ok
version_api | ok
installed_addons | File editor (5.3.3), Mosquitto broker (6.0.1), Zigbee2mqtt (1.24.0-1), Zigbee2mqtt (1.24.0-1), Samba share (9.5.1), Studio Code Server (4.1.0), MPD (1.7.0), Terminal & SSH (9.3.0), ESPHome (2022.2.6), MariaDB (2.4.0), VLC (0.1.3)

</details>

<details><summary>Lovelace</summary>

dashboards | 1
-- | --
resources | 6
views | 5
mode | yaml

</details>

**Comment by Efendis** [comment]
Hello, did you find a problem?

**Comment by MartinHjelmare** [comment]
I've relayed the information to the OS team. I don't know more at the moment unfortunately.

**Comment by realthk** [comment]
I'm getting the exact same errors more and more recently.
Home Assistant OS 7.6 Generic x86-64, just upgraded to 20220504.0, but it has happened several times with 20220503.x versions also. 
I wonder is it related to HassOS version or VLC 0.1.3. 
I'll try to install an older version of VLC

**Comment by Efendis** [comment]
I confirm, for the second day in a row I have to reload the addon!

**Comment by acburnett** [comment]
Im having the same issue here on a supervised debian install. I dont think it started happening until the supervisor 2022.05.0 update a few days ago. I dont know how to downgrade the supervisor or id try.

**Comment by realthk** [comment]
> I dont think it started happening until the supervisor 2022.05.0 update a few days ago. I dont know how to downgrade the supervisor or id try.

I also could not test downgrading VLC addon yet as I don't have old enough backup, though checking [its last minor changes](https://github.com/home-assistant/addons/commits/master/vlc), I'm not sure it could be the reason (unless of course, VLC itself has changed, and a new version of it was installed during the addon docker is being built), 
but I do not suspect Supervisor 2022.05.0 as it was released only 4 days ago, and the problem has occured on my machine earlier than that.

**Comment by Efendis** [comment]
Home Assistant OS 8.0 The problem is resolved!

**Comment by Efendis** [comment]
Yes, OS 10.2, the problem reoccurs. usb stick Doesn't work either.

**Comment by acburnett** [comment]
The fix is merged into supervisor 2023.06.2 but it isnt realeased to stable branch yet.

**Comment by johnboiles** [comment]
Great news that a fix is coming!
